src/formatDate.js

```diff
@@ -50,7 +50,31 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const oldSeparator = fromFormat.pop();
+  const newSeparator = toFormat.pop();
```

Comment:
In this task it doesn't really matter, but remember that `pop()` is a mutating method.
And in this case it mutates the function arguments.
You need to avoid this if it's not mentioned directly in the task description.

Reply:
Thanks for noticing and explaining 👍  It's quite important thing, dunno how i missed that 😅